tox.ini

```diff
@@ -8,7 +8,7 @@
 ; * nodeps - avoid installing any dependencies apart from testing tools.
 ; * withcov - run the tests wrapped using the coverage.
 ; * nocov - run the tests directly, without using the coverage wrapper.
-; * wheel - build the wheel distribution
```

Comment:
This wheel target should not have been removed.

Reply:
tox now builds the wheel before running any stage - it doesn't need a separate target to do it anymore

Reply:
@graingert This affects the release process.  You and Adiroiban are not doing the current release (I am), so this is out of your scope to review, although technically the Twisted review process does not forbid this.

Reply:
I think that we need the help from @glyph or @exarkun to weight their views on this.

I stand by my review and I think that @graingert did an excellent work
and this PR is a big improvement for the Twisted release process.

-----------

I know that the review process for this PR is on the grey area.

The code was created by @graingert in a fork.
To review and tests it, I have created this PR in a branch on the main repo.
So by creating this PR I was doing the review. 

----------

I might be wrong and I am not against reverting this PR if required.

Reply:
Another argument for keeping this PR (inspired by Jean Paul, see irc logs https://freenode.logbot.info/twisted-dev/20210201#c6744337).

This argument represent only my views  and I am not representing JP or the Twisted community here :)
 
This PR is part of a long term goal of automating the release process.
The idea is that with this PR, you only need to create a tag, and the wheels are published by themselves :)
The release manager will not have to spend time on manually creating and publish the wheels.

The current release is in process since a very long time and there was not much progress.
I was feeling that by just following the old release process, we will continue to have delays.

So in parallel with the current release, some Twisted contributors (including myself) have tried to find a way to make the release work.

Cheers

Reply:
You've done the following in the past few months:
• Made changes to the release automation
• Made changes to the release documentation
• Get people who are not doing the release, and have no track record doing Twisted releases to approve your commits

This behavior is very aggressive on your part.

While I've been a bit slow at this, on my part, I have gone out of my way to email Glyph and Amber, and
try to make sure that they were OK with me doing this release.  Unfortunately, I have not received any feedback from Amber, but Glyph gave me permission to do the release, so I have been proceeding with this.

I'm sorry that I have been a bit slow at releasing Twisted fast enough for you, but this is my first time doing this, and with the last issue fixed, I will do the release.

Reply:
I am not in a hurry to see a release or have it on PyPI.
For my personal project I use a fork and an private package index.

If you want to revert this, feel free to do it.

The changes that I made were not done in a hurry and I asked for feedback over IRC, beside the normal GitHub PR.

---------

I have worked with Thomas in the past on ldaptor and I know he is a good release manager and up to date with latest best practices in terms of Python package management and release process. 

I consider that he is very competent to do and review release tasks.

My suggestion is to move this conversation over the mailing list so that more people could weight their opinion. 

Thanks!

Reply:
I would like to revert all these changes you have done to the release documentation and
process.

However, it looks like someone (probably you?)  has made a change to the GitHub Twisted repository, so
that a PR can only be merged to trunk, if it has been approved by someone who has write access
to the Twisted repository.   As someone with write access to the Twisted repository, in the past (as recent as summer 2020), it was possible
for me to get reviews approved by people who did not have write access to the repository.
This is not the case any more.
In private e-mail, Glyph said that he did not make this change to the GitHub repository, but he is in favor of keeping this configuration.  So I will respect that.

If I start reverting these changes, I will need to get review approval from another person with write access to the Twisted repository.  This will further delay the release, which I am not in favor of doing.

So in the interest of moving things along with the release, I will not be reverting these changes,
but I will express my disapproval.

I appreciate your enthusiasm for the Twisted project and all that you have done over the years, such as trying
to revive the Project Leadership Committee.
Unfortunately, in this area, I find that your recent behavior is very aggressive,
although it is entirely legal via the Twisted development process.

Reply:
Craig, can I suggest that you take the contributions of adiroiban and graingert to be in good faith?

Twisted is a group effort.  Right now, I see three people who want to help Twisted in this area.  I think that can be a great thing as long as we are working towards a common goal and not at cross purposes.

A lot of Twisted developers engage on IRC and on the mailing list which helps them come to a common goal and synchronize their efforts.

From your comments, I infer that you've had some private conversations with some people - but not with adiroiban or graingert.  Do you think you could start having some public conversations about what your goals and activities are?  I would suggest starting a thread on the Twisted mailing list rather than continuing to comment on this PR (which is public but obscure enough that a lot of people are going to miss out on the comments - as you yourself missed out on this PR when it was first submitted).  I think that until all interested parties are productively communicating with each other, it's going to be very hard to resolve this to everyone's satisfaction.

I hope you take this comment in the spirit it is given - as an attempt to constructively work through some obstacles that have arisen from lack of a shared goal and coordinated efforts towards that goal - not as an attack or any sort of aggression towards you.

Reply:
@exarkun thanks for the feedback.  Yes, I believe that adiroiban and graingert have been doing things in good
faith, however, the way it has been done comes across as aggressive, and is unappreciated by me.

I think that I have expressed my views in this thread, in in other PR's, so I don't feel a need to
open another thread on the mailing list.

In terms of private conversations, I have reached out to Glyph and Amber (who did the last release).
I tried to be respectful of Amber's role, and make sure that I was not offending anyone by stepping up and trying to release Twisted.  Unfortunately, Amber has not responded to any of my emails, but Glyph has.

With respect to the release, for now, I am focusing on the last technical issue, and then cutting the release.

I think some of the root causes behind this problems we are seeing in this thread are due to the lack of an active Project Leadership Committee.  If there is anything you can do to help in that regard, especially in terms of answering these questions which I raised on the mailing list: https://twistedmatrix.com/pipermail/twisted-python/2021-January/065405.html , that would be great.

Reply:
> I think some of the root causes behind this problems we are seeing in this thread are due to the lack of an active Project Leadership Committee. If there is anything you can do to help in that regard, especially in terms of answering these questions which I raised on the mailing list: https://twistedmatrix.com/pipermail/twisted-python/2021-January/065405.html , that would be great.

There's a lot going on in this discussion thread and I'll try to write a more substantive message later, but I just wanted to address this specific detail:

The Project "Leadership" Committee is perhaps unfortunately named.  This implies that the question you were asking about "roles" was not what I thought it was — about division of responsibilities within the group — but instead about what the group does as a whole.

So: the committee's sole responsibility is to *suggest that expenditures be approved* to the Software Freedom Conservancy, based on existing community consensus.  (I can't say "approve expenditures" because it doesn't have the unilateral ability to do so, its oversight is subject to the authority of the SFC itself.)  A better name for the group would be the Twisted Project Budgetary Oversight Committee.  Much as the Python Software Foundation does not manage the day-to-day operations of the Python core development team, even a highly efficient PLC would not have any role in addressing issues like this one, except in that it might hire someone to do the work.

So there's authority coming to save us here, we need to build a community consensus through discussion of the issues on the merits.